src/service/MailboxService.js

```diff
@@ -46,3 +46,12 @@ export const deleteMailbox = async(id) => {
 
 	await axios.delete(url)
 }
+export async function patchMailbox(accountId, mailboxId, data) {
+	const url = generateUrl('/apps/mail/api/accounts/{accountId}/mailboxes/{mailboxId}', {
```

Comment:
```suggestion
	const url = generateUrl('/apps/mail/api/mailboxes/{mailboxId}', {
```

not 100% sure but I think you're using an outdated URL format. Check `appinfo/routes.php` for comparison.